### DISCUSSION TITLE
When will Muse Sounds be available?

### DISCUSSION THREAD

**Discussion by timothydaly1** [musescore/MuseScore#13730] [Q&A] [answered]
The title basically says it all. I've been playing around with the alpha version, and am loving the new UI. I was wondering when Muse Sounds will be released to the public. Will it be released with v4, or potentially before that?

**Comment by KaiVinter** [comment, marked as answer]
I'm not sure if you're on Discord, because all the conversation related to Muse Sounds is happening there. But in short, it will be available when the Beta 1 comes out, which seems pretty soon. There's no date, since the development of a piece of software can't have definite dates because new bugs appear, but we can estimate either this week or the next. But don't quote me on that.
